runtime/bcutil/ClassFileOracle.cpp

```diff
@@ -81,8 +81,13 @@ ClassFileOracle::KnownAnnotation ClassFileOracle::_knownAnnotations[] = {
 		{SCOPED_SIGNATURE, sizeof(SCOPED_SIGNATURE)},
 #undef SCOPED_SIGNATURE
 #endif /* JAVA_SPEC_VERSION >= 16 */
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+#define NOTCHECKPOINTSAFE_SIGNATURE "Lopenj9/internal/criu/NotCheckpointSafe;"
+		{NOTCHECKPOINTSAFE_SIGNATURE, sizeof(NOTCHECKPOINTSAFE_SIGNATURE)},
+#undef NOTCHECKPOINTSAFE_SIGNATURE
 		{0, 0}
```

Comment:
This needs to be outside the ifdef - it terminates the table. Non-CRIU builds will fail.